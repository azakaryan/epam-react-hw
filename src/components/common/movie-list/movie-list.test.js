import React from 'react';
import { shallow } from 'enzyme';

import expectEnzymify from 'expect';
import enzymify from 'expect-enzyme';
import MovieList from './movie-list';

expectEnzymify.extend(enzymify());

describe('MovieList Component', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<MovieList debug />);

    expect(component).toMatchSnapshot();
  });

  // it('renders a MovieList element', () => {
  //   const wrapper = shallow(<MovieList debug />);
  //   // TODO how to use expect for both cases.
  //   expectEnzymify(wrapper).toBeA('div');
  // });

  it('should render movies correctly with given data', () => {
    const component = shallow(<MovieList/>);
    expect(component).toMatchSnapshot();
  });
});
