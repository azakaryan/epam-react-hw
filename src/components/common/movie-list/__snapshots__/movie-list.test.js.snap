// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieList Component should render correctly in "debug" mode 1`] = `""`;

exports[`MovieList Component should render movies correctly with given data 1`] = `""`;
